### PR TITLE
docs: Added an info panel reg. etcd.clusterDomain

### PR DESCRIPTION
**Pull request by balaharish7**
Added an info panel reg. "etcd.clusterDomain" in the [Installing OpenEBS](https://openebs.io/docs/quickstart-guide/installation) documentation.

This PR was created in regards with [#3741](https://github.com/openebs/openebs/issues/3741#issuecomment-2175478994)

### CODE REVIEW

**Review by tiagolobocastro — COMMENTED** [review]
hmm don't see any changes regarding the cluster domain here?

File: docs/main/quickstart-guide/installation.md [hunk 1]
```diff
@@ -66,6 +66,10 @@ OpenEBS provides several options to customize during installation such as:
 - Specifying the directory where hostpath volume data is stored or
 - Specifying the nodes on which OpenEBS components should be deployed and so forth. 
 
+:::info
+See [here](https://github.com/openebs/openebs/blob/main/charts/README.md#values) for more configurations.
```
Comment (avishnu) [inline docs/main/quickstart-guide/installation.md:70]: for configurable options.

File: docs/main/faqs/faqs.md [hunk 2]
```diff
@@ -648,6 +648,18 @@ Now that `0000:01:00.0` is bound to `vfio-pcie`, you can create your pools with
 
 [Go to top](#top)
 
+### How do I install Replicated PV Mayastor on a Kubernetes cluster with a custom domain?
+
+You must configure etcd for Replicated PV Mayastor during installation if the domain name of your Kubernetes cluster is not 'cluster.local'.
```
Comment (avishnu) [inline docs/main/faqs/faqs.md:653]: "You must configure etcd" seems incorrect.
Need to be rephrased differently

Comment (balaharish7) [reply docs/main/faqs/faqs.md:653]: Re-writing as:
If the domain name of your Kubernetes cluster is not 'cluster.local', you have to specify the custom domain during installation as follows:"

**Review by avishnu — APPROVED** [review]

**Pull request merged**